pkg/fixer/reporter.go

```diff
@@ -50,9 +50,7 @@ func (r *PrettyReporter) ReportConflicts(fixReport *Report) error {
 	if len(roots) > 0 {
 		fmt.Fprintln(r.outputWriter, "Source file conflicts:")
 
-		i := 0
-
-		for _, rootKey := range roots {
```

Comment:
Looks like this just didn't work before? `i` was never increased...

Reply:
Indeed. Not too disastrous though as it was just for printing an extra newline between groups.